module-system/module-implementations/sov-prover-incentives/src/lib.rs

```diff
@@ -8,7 +8,9 @@ mod tests;
 mod query;
```

Comment:
Let's add the following lints:
```
#![deny(missing_docs)]
#![doc = include_str!("../README.md")]
```
So it will fail compilation if a public doc is missing and add README as crate level doc.